tests/pot/config.yml

```diff
@@ -0,0 +1,16 @@
+model:
+  model_ir_path : 'model_ir'
+
+task_type: QuestionAnswering

```

Comment:
Can we check the task type from the model object?

Reply:
Done